perun-core/src/test/java/cz/metacentrum/perun/core/bl/VosManagerBlImplUnitTest.java

```diff
@@ -0,0 +1,90 @@
+package cz.metacentrum.perun.core.bl;
+
+import cz.metacentrum.perun.core.api.*;
+import cz.metacentrum.perun.core.api.exceptions.UserNotExistsException;
+import cz.metacentrum.perun.core.blImpl.VosManagerBlImpl;
+import cz.metacentrum.perun.core.implApi.VosManagerImplApi;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.*;
+
+public class VosManagerBlImplUnitTest {
+	private VosManagerBlImpl vosManagerBl;
+	private VosManagerImplApi vosManagerImpl = mock(VosManagerImplApi.class);
+	private PerunSession sess = mock(PerunSession.class);
+	private Candidate candidate1 = mock(Candidate.class);
+	private Candidate candidate2 = mock(Candidate.class);
+	private Vo vo = mock(Vo.class);
+	private Group group = mock(Group.class);
+	private VosManagerBlImpl vosManagerBlSpy;
+
+	@Before
+	public void setUp() {
+		vosManagerBl = new VosManagerBlImpl(vosManagerImpl);
+		vosManagerBl.setPerunBl(mock(PerunBl.class, RETURNS_DEEP_STUBS));
+		vosManagerBlSpy = spy(vosManagerBl);
+
```

Comment:
Please, try to keep the code as much formatted as possible. Try to avoid these empty lines.